src/EventLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import useStyles from "./styles/EventLinkStyles";
+
+export default function EventLink({ eventtitle }) {
+  const classes = useStyles();
+
+  return ( 
+    
+    <div className={classes.root} > 
+    <div className={classes.link}>
+    {eventtitle}
```

Comment:
You could probably get an extension for your IDE that automatically fixes indentation for you. I use one called "Prettier - Code Formatter" for Visual Studio Code and have it configured to run everytime I save a file